app/dev_mode/views.py

```diff
@@ -74,7 +75,7 @@ def extract_eq_id_and_form_type(schema_name):
 
 
 def create_payload(user, exp_time, eq_id, period_str, period_id, form_type, collection_exercise_sid, ref_p_start_date,
```

Comment:
This is used for integration tests and we should really be sticking to the pattern of passing in whats needed as opposed to a wrapping object